src/codeu/chat/database/Database.java

```diff
@@ -0,0 +1,91 @@
+package codeu.chat.database;
+
+import codeu.chat.common.Conversation;
+import codeu.chat.common.Message;
+import codeu.chat.common.User;
+import codeu.chat.util.Uuid;
+import com.mongodb.MongoClientURI;
+import org.bson.Document;
+
+import com.mongodb.MongoClient;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+
+public class Database {
+    public MongoClient mongoClient;
+    public MongoDatabase database;
+    public MongoCollection<Document> users;
+    public MongoCollection<Document> messages;
+    public MongoCollection<Document> conversations;
+
+    private boolean clearDatabase = false;
+    private String username = "User";
+    private String password = "SuperSecretPassword";
+    private String databasePath = "TestDatabase";
+    private String URI = "mongodb://" + username + ":" + password + "@codeu-shard-00-00-kskbs.mongodb.net:27017,codeu-shard-00-01-kskbs.mongodb.net:27017,codeu-shard-00-02-kskbs.mongodb.net:27017/" + databasePath + "?ssl=true&replicaSet=CodeU-shard-0&authSource=admin";
```

Comment:
I decided to keep it how it is, mainly because it allows the person looking at it to easily see where the parameters go into the URI.